index.js

```diff
@@ -17,14 +22,53 @@ async function run() {
     const dockerLabels = core.getInput('docker-labels', { required: false });
     const command = core.getInput('command', { required: false });
 
-    // Parse the task definition
-    const taskDefPath = path.isAbsolute(taskDefinitionFile) ?
-      taskDefinitionFile :
-      path.join(process.env.GITHUB_WORKSPACE, taskDefinitionFile);
-    if (!fs.existsSync(taskDefPath)) {
-      throw new Error(`Task definition file does not exist: ${taskDefinitionFile}`);
+    //New inputs to fetch task definition 
+    const taskDefinitionArn = core.getInput('task-definition-arn', { required: false }) || undefined;
+    const taskDefinitionFamily = core.getInput('task-definition-family', { required: false }) || undefined;
+    const taskDefinitionRevision = Number(core.getInput('task-definition-revision', { required: false })) || null;
+
+    let taskDefPath;
+    let taskDefContents;
+    let describeTaskDefResponse;
+    let params;
+    
+    if (taskDefinitionFile) {
+      core.warning("Task definition file will be used.");
+      taskDefPath = path.isAbsolute(taskDefinitionFile) ?
+        taskDefinitionFile : 
+        path.join(process.env.GITHUB_WORKSPACE, taskDefinitionFile);
+      if (!fs.existsSync(taskDefPath)) {
+        throw new Error(`Task definition file does not exist: ${taskDefinitionFile}`);
+      }
+      taskDefContents = require(taskDefPath);
+    } else if (taskDefinitionArn || taskDefinitionFamily || taskDefinitionRevision) {
+      if (taskDefinitionArn) {
+        core.warning("The task definition arn will be used to fetch task definition");
+        params = {taskDefinition: taskDefinitionArn};
+      } else if (taskDefinitionFamily && taskDefinitionRevision) {
+        core.warning("The latest revision of the task definition family will be provided");
```

Comment:
Warning needs to be changed to "The **specified** revision of the task definition family will be used to fetch task definition". 

*specified instead of latest* ?